netmanager-app/core/urls.tsx

```diff
@@ -6,3 +6,4 @@ export const USERS_MGT_URL = `${BASE_API_URL}/users`;
 export const DEVICES_MGT_URL = `${BASE_API_URL}/devices`;
 export const SITES_MGT_URL = `${BASE_API_URL}/devices/sites`;
 export const ANALYTICS_MGT_URL = `${BASE_API_URL}/analytics`;
+export const EXPORT_DATA_URL = `${BASE_API_URL}/analytics/data-download`;
```

Comment:
We already have the base url for analytics endpoints set up right above there called ANALYTICS_MGT_URL, simply use that and just add/data-download while defining the api @danielmarv